Makefile

```diff
@@ -40,7 +67,7 @@ backup_database:
 	
 restore_database:
 	@docker exec -i lantern-back-end_postgres_1 pg_restore --clean -U lantern -d lantern < $(file)
-	@echo "Database was restored from $(file)"
```

Comment:
Was this delete a mistake?

Reply:
Oh weird, yeah it must have been a mistake, I will add this back in now!